Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -16,6 +16,8 @@ open class ChatChannelVC: _ViewController,
     /// Controller for observing data changes within the channel.
     open var channelController: ChatChannelController!
 
+    public var initialMessage: ChatMessage?
```

Comment:
Where are we using this?

Reply:
Nope, this was a left over from a previous implementation 👍